products/models.py

```diff
@@ -0,0 +1,49 @@
+from django.db import models
+
+# Create your models here.
+class Category(models.Model):
+    name        = models.CharField(max_length=45)
+    description = models.CharField(max_length=200)
+    image_url   = models.CharField(max_length=500)
+
+    class Meta:
+        db_table = 'categories'
+
+class Sub_category(models.Model):
```

Comment:
```suggestion
class SubCategory(models.Model):
```
class 이름은 _ 대신에 대문자로 단어를 구분합니다.